apps/ledger-live-mobile/src/newArch/features/Web3Hub/components/Disclaimer/index.tsx

```diff
@@ -35,7 +35,7 @@ export default function Disclaimer({
 
       {description ? (
         <Flex mt={6}>
-          <Text fontSize={14} lineHeight={"22px"} color="smoke">
+          <Text numberOfLines={12} fontSize={14} lineHeight={"22px"} color="smoke">
```

Comment:
Does it impact Disclaimer or just a small modification ?

Reply:
While navigating on the mocks I saw a big issue with descriptions too long, so just included the fix here